modules/apiserversource-yaml.adoc

```diff
@@ -236,10 +273,34 @@ Data,
 [id="delete-apiserversource-yaml_{context}"]
 == Deleting the ApiServerSource
 
-You can delete the ApiServerSource, trigger, service, service account, cluster role, and cluster binding created in this guide by entering the following `oc` commands:
+. Delete the trigger by entering the following command:
++
+
+[source,terminal]
+----
+$ oc delete -f trigger.yaml
+----
+
+. Delete the service by entering the following command:
++
+
+[source,terminal]
+----
+$ oc delete -f service.yaml
+----
+
+. Delete the event source by entering the following command:
++
+
+[source,terminal]
+----
+$ oc delete -f k8s-events.yaml
+----
+
+. Delete the service account, cluster role, and cluster binding created in this guide by entering the following command:
```

Comment:
s/and cluster binding created in this guide by entering the following command:/and cluster binding:/